README.md

```diff
@@ -133,23 +133,4 @@ docker-compose -f docker-compose.test.yml up --build --exit-code-from test
 ~~~
 
 ### Developing / Debugging
-Build and start your container in "developer mode"
-~~~bash
-./scripts/devel-compose build --no-cache vmaas_webapp
-./scripts/devel-compose up vmaas_webapp
-~~~
-
-switch inside of the container
-~~~bash
-./scripts/devel-compose exec vmaas_webapp bash
-~~~
-
-now your local git directory is mounted under `/git` in the container so any change
-you make you can immediatelly test.
-
-```[root@4cb6b50d0cb6 git]# python3 ./app.py```
-
-Note that by default container does NOT run the application (so you can run your own modificationtion)
-so if you want to run "original" (unmodified) application use
-
-```[root@4cb6b50d0cb6 git]# /app/entrypoint.sh```
+You can build and start your container in ["developer mode"](doc/build_containers.md).
```

Comment:
Ok, but I would rename file to `developer_mode.md`. It's better regarding to its content. Also quotation marks are unnecessary.